src/fides/api/alembic/migrations/versions/3038667ba898_update_system_legal_basis_fields.py

```diff
@@ -0,0 +1,158 @@
+
+import sqlalchemy as sa
+from alembic import op
+from loguru import logger
+from sqlalchemy.orm.session import Session
+from sqlalchemy.sql import text
+
+# revision identifiers, used by Alembic.
+revision = "3038667ba898"
+down_revision = "507563f6f8d4"
+branch_labels = None
+depends_on = None
+
+legal_basis_transfers_upgrades: Dict[str, str] = {
+    "Adequacy decision": "Adequacy Decision",
+    "Standard contractual clauses": "SCCs",
+    "Binding corporate rules": "BCRs",
+}
+
+legal_basis_transfers_downgrades: Dict[str, str] = {
+    value: key for key, value in legal_basis_transfers_upgrades.items()
+}
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+    conn = session.connection()
+    bind = op.get_bind()
+
+    #############################
+    # legal_basis_for_transfers #
+    #############################
+    statement_transfers = text(
+        """SELECT distinct legal_basis_for_transfers, id
+FROM ctl_systems"""
+    )
+    existing_values_transfers = conn.execute(statement_transfers)
+
+    # drop column since we will repopulating later
+    op.drop_column(
+        "ctl_systems",
+        "legal_basis_for_transfers",
+    )
+
+    # add column back, but make it multi-value
+    op.add_column(
+        "ctl_systems",
+        sa.Column(
+            "legal_basis_for_transfers",
+            type_=sa.ARRAY(sa.String()),
+            server_default="{}",
+            nullable=True,
+        ),
+    )
+
+    # data migration - updates to some values since allowable values enum has changed
+    # (enum is not enforced/specified on the DB, just API-level)
+
+    for row in existing_values_transfers:
+        existing_value = row[0]
+        updated_value = legal_basis_transfers_upgrades.get(
+            existing_value, existing_value
+        )
+        updated_value = (
+            [updated_value] if updated_value else []
+        )  # wrap in a list because column is now array type
+        bind.execute(
+            text(
+                "UPDATE ctl_systems SET legal_basis_for_transfers = :updated_basis WHERE id = :id"
+            ),
+            {"updated_basis": updated_value, "id": row[1]},
+        )
+
+    #############################
+    # legal_basis_for_profiling #
+    #############################
+    statement_profiling = text(
+        """SELECT distinct legal_basis_for_profiling, id
+FROM ctl_systems"""
+    )
+    existing_values_profiling = conn.execute(statement_profiling)
+
+    op.drop_column(
+        "ctl_systems",
+        "legal_basis_for_profiling",
+    )
+    op.add_column(
+        "ctl_systems",
+        sa.Column(
+            "legal_basis_for_profiling",
+            type_=sa.ARRAY(sa.String()),
+            server_default="{}",
+            nullable=True,
+        ),
+    )
+
+    # data migration - wrap values in a list
+    for row in existing_values_profiling:
+        existing_value = row[0]
+        updated_value = legal_basis_transfers_upgrades.get(
```

Comment:
taking another look 👀 

there was this copy paste issue but that shouldn't have had a practical effect.  The profiling value wouldn't have existed in the legal basis for transfers dictionary, so we would have used the original profiling value and added it to the list